Assets/GLTF/Scripts/GLTFLoader.cs

```diff
@@ -252,21 +353,110 @@ protected virtual GameObject CreateMeshPrimitive(MeshPrimitive primitive)
 
 					tangents = primitive.Attributes.ContainsKey(SemanticProperties.TANGENT)
 						? primitive.Attributes[SemanticProperties.TANGENT].Value.AsTangentArray()
+						: null,
+
+					boneWeights = primitive.Attributes.ContainsKey(SemanticProperties.Weight(0)) && primitive.Attributes.ContainsKey(SemanticProperties.Joint(0))
+						? CreateBoneWeightArray(primitive.Attributes[SemanticProperties.Joint(0)].Value.AsVector4Array(), primitive.Attributes[SemanticProperties.Weight(0)].Value.AsVector4Array(), vertexCount)
```

Comment:
Could we try to avoid a line this long?

Reply:
Not easily, since it would actually hurt performance to do this work early. I did new line it, but unfortunately I don't think it reads any better.